shell/platform/linux/public/flutter_linux/fl_value.h

```diff
@@ -63,6 +65,7 @@ typedef enum {
   FL_VALUE_TYPE_UINT8_LIST,
   FL_VALUE_TYPE_INT32_LIST,
   FL_VALUE_TYPE_INT64_LIST,
+  FL_VALUE_TYPE_FLOAT32_LIST,
```

Comment:
@stuartmorgan we need to maintain ABI with this library, right?

This change would cause something compiles against an older version of libflutter_linux_gtk.so to crash if using `FL_VALUE_TYPE_FLOAT_LIST` or above. `FL_VALUE_TYPE_FLOAT32_LIST` really should be added to the end of the enum.

Reply:
Ah, yeah. I didn't catch that this is part of the pubic interface. Thanks!

Reply:
This should be fixed.